String/Lower.js

```diff
@@ -12,15 +12,12 @@ const lower = (str) => {
     throw new TypeError('Invalid Input Type')
   }
 
-  let lowerString = ''
+  const lowerString = str
```

Comment:
```suggestion
  return str
```
No need to declare a new variable, just return the function from here.

Reply:
Ok.